src/python/grpcio/commands.py

```diff
@@ -262,6 +262,10 @@ def new_compile(obj, src, ext, cc_args, extra_postargs, pp_opts):
                     extra_postargs = [
                         arg for arg in extra_postargs if not '-std=c++' in arg
                     ]
+                elif src.endswith('.cc') or src.endswith('.cpp'):
```

Comment:
nit: should we use src.endswith instead of  `src[-2:]` on the line above? (for consistency)

Reply:
Done. Good idea!